quiz.py

```diff
@@ -2,5 +2,51 @@
 File: quiz.py
 Author: Name
 Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Description: A Multiple choice game that calculates the score
+"""
+print("Multiple-Question Quiz Game (NBA)")
+
+correct_answers = 0
+
+print("1. Who scored the most career points in the NBA?")
+print("(a) Michael Jordan")
+print("(b) Lebron James")
+print("(c) Kareem Abdul-Jabbar")
+
+answer_1 = input("Answer:")
+
+if answer_1.lower() == "b":
+    print("Correct!")
+    correct_answers += 1
+else:
+    print ("Incorrect")
+
+print("2. Who scored the most 3 career points in the NBA?")
+print("(a) Stephen Curry")
+print("(b) Ray Allen")
+print("(c) Kyle Korver")
+
+answer_2 = input("Answer:")
+
+if answer_2.lower() == "a":
+    correct_answers += 1
+    print("Correct!")
+else:
+    print("Incorrect!")
+
+print("3. Who scored the most points in a single NBA game?")
+print("(a) Lebron James")
+print("(b) Kareem Abdul-Jabbar")
+print("(c) Kobe Bryant")
+
+answer_3 = input("Answer:")
+
+if answer_3.lower() == "b":
+    correct_answers += 1
+    pri
+
+# Calculate the percentage score
+score_percentage = (correct_answers / 3) * 100
+
+# Print the quiz completion message with the score
+print("Quiz complete! You answered {} out of 3 questions correctly. Your score is {:.1f}%".format(correct_answers, score_percentage))
```

Comment:
I see you are using f-strings for your output. This is not very readable. I see the first `{}` but then I have to go to the very end of your statement to see what is supposed to be in its place (value of `correct_answers`). It's obscure and doesn't make for clean, easily-understood code.

If you are insistent on using f-strings (versus, say, standard concatenation) can you use another style that allows the output statement to be understood with inline variables in the string literal?